### PR TITLE
Move issue created function to cloud run

### PR DESCRIPTION
**Pull request by halamix2**
**Description**

Changes proposed in this pull request:

- Move issue created function to cloud run

**Related issue(s)**

/kind feature
/area tools
/retest

### CODE REVIEW

File: configs/terraform/modules/github-webhook-gateway/github-webhook-gateway.tf [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      service_account_name = google_service_account.github_webhook_gateway.email
+      containers {
+        image = "europe-docker.pkg.dev/kyma-project/prod/test-infra/ko/github-webhook-gateway:v20230802-a960e47d"
+        env {
+          name  = "PROJECT_ID"
+          value = var.gcp_project_id
+        }
+        env {
+          name  = "COMPONENT_NAME"
+          value = "github-webhook-gatewaydge"
```
Comment (dekiel) [inline configs/terraform/modules/github-webhook-gateway/github-webhook-gateway.tf:43]:
```suggestion
          value = "github-webhook-gateway"
```

File: configs/terraform/modules/github-webhook-gateway/github-webhook-gateway.tf [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        }
+        env {
+          name  = "APPLICATION_NAME"
+          value = "github-webhook-gatewaydge"
```
Comment (dekiel) [inline configs/terraform/modules/github-webhook-gateway/github-webhook-gateway.tf:47]:
```suggestion
          value = "github-webhook-gateway"
```

File: configs/terraform/modules/slack-message-sender/slack-message-sender.tf [hunk 3]
```diff
@@ -53,11 +53,15 @@ resource "google_cloud_run_service" "slack_message_sender" {
...
         }
         env {
           name  = "SLACK_RELEASE_CHANNEL_ID"
-          value = "C01KKPXCPK8"
+          value = "C01KKPXCPK8" # kyma-skr-release
+        }
+        env {
+          name  = "SLACK_TEAM_CHANNEL_ID"
+          value = "C01LGCBS196" # kyma-team
         }
```
Comment (dekiel) [inline configs/terraform/modules/slack-message-sender/slack-message-sender.tf:65]: Can we have this env names more meaningfull? Something like:
PROW_DEV_NULL_SLACK_CHANNEL_ID
RELEASE_SLACK_CHANNEL_ID
KYMA_TEAM_SLACK_CHANNEL_ID

File: development/gcp/cloud-run/github-webhook-gateway/github-webhook-gateway.go [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+	"github.com/kyma-project/test-infra/development/gcp/pkg/cloudfunctions"
+	crhttp "github.com/kyma-project/test-infra/development/gcp/pkg/http"
+	"github.com/kyma-project/test-infra/development/gcp/pkg/pubsub"
+	toolsclient "github.com/kyma-project/test-infra/development/github/pkg/client"
+	"github.com/kyma-project/test-infra/development/types"
+)
+
+const (
+	SapToolsGithubURL = "https://github.tools.sap/"
+)
```
Comment (dekiel) [inline development/gcp/cloud-run/github-webhook-gateway/github-webhook-gateway.go:21]: This const is already defined in a package in our repo. https://github.com/kyma-project/test-infra/blob/fbdfd267f1adedae6c230c006c8f0bf21e23c294/development/github/pkg/client/client.go#L23C50-L23C50

**Review by dekiel — CHANGES_REQUESTED** [review]

File: development/gcp/cloud-run/github-webhook-gateway/github-webhook-gateway.go (hunk 4 above)
Comment (halamix2) [reply development/gcp/cloud-run/github-webhook-gateway/github-webhook-gateway.go:21]: This const is not needed anymore, I'll remove it

**Review by dekiel — APPROVED** [review]

**Pull request merged**